### PR TITLE
add eospac-wrapper to install manifest

### PR DESCRIPTION
**Pull request by mauneyc-LANL**
## PR Summary

Addresses further header includes given in #227 . This time is for `eospac_wrapper.hpp`, which I for some reason thought was a privately used by a binary and not a library archive.

The following is an example of an install:
```bash
$ cmake --install .
-- Install configuration: ""
-- Installing: <install-prefix>/include/eospac-wrapper/eospac_wrapper.hpp
-- Installing: <install-prefix>/lib/libeospac-wrapper.a
-- Installing: <install-prefix>/lib/cmake/singularity-eos/eospac-wrapperTargets.cmake
-- Installing: <install-prefix>/lib/cmake/singularity-eos/eospac-wrapperTargets-noconfig.cmake
-- Installing: <install-prefix>/lib/libsingularity-eos.a
-- Installing: <install-prefix>/lib/cmake/singularity-eos/singularity-eosTargets.cmake
-- Installing: <install-prefix>/lib/cmake/singularity-eos/singularity-eosTargets-noconfig.cmake
-- Installing: <install-prefix>/include/singularity-eos/closure/mixed_cell_models.hpp
-- Installing: <install-prefix>/include/singularity-eos/base/fast-math/logs.hpp
-- Installing: <install-prefix>/include/singularity-eos/base/robust_utils.hpp
-- Installing: <install-prefix>/include/singularity-eos/base/root-finding-1d/root_finding.hpp
-- Installing: <install-prefix>/include/singularity-eos/base/variadic_utils.hpp
-- Installing: <install-prefix>/include/singularity-eos/base/math_utils.hpp
-- Installing: <install-prefix>/include/singularity-eos/base/constants.hpp
-- Installing: <install-prefix>/include/singularity-eos/base/eos_error.hpp
-- Installing: <install-prefix>/include/singularity-eos/base/sp5/singularity_eos_sp5.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_variant.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/singularity_eos.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_stellar_collapse.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_ideal.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_spiner.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_davis.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_gruneisen.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_builder.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_jwl.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/modifiers/relativistic_eos.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/modifiers/scaled_eos.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/modifiers/ramps_eos.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/modifiers/shifted_eos.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/modifiers/eos_unitsystem.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_base.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/eos_eospac.hpp
-- Up-to-date: <install-prefix>/include/singularity-eos/eos/singularity_eos.hpp
-- Up-to-date: <install-prefix>/include/singularity-eos/closure/mixed_cell_models.hpp
-- Installing: <install-prefix>/include/singularity-eos/eos/singularity_eos.mod
-- Installing: <install-prefix>/include/singularity-eos/eos/singularity_eos_types.mod
-- Installing: <install-prefix>/lib/cmake/singularity-eos/singularity-eosConfig.cmake
-- Installing: <install-prefix>/lib/cmake/singularity-eos/FindEOSPAC.cmake
```

Build with:

```bash
cmake .. -DSINGULARITY_USE_EOSPAC=ON -DSINGULARITY_BUILD_TESTS=ON -DSINGULARITY_TEST_SESAME=ON -DSINGULARITY_BUILD_CLOSURE=ON -DSINGULARITY_USE_FORTRAN=ON -DCMAKE_INSTALL_PREFIX=./dumpinstall
```

So use downstream is:

```c++
#include "eospac-wrapper/eospac_wrapper.hpp"
```

These fixes has lead me to believe a more streamlined install procedure is necessary. I will try to incorporate these into #221 

## PR Checklist

- N/A Adds a test for any bugs fixed. Adds tests for new features.
- N/A Format your changes by using the `make format` command after configuring with `cmake`.
- N/A Document any new features, update documentation for changes made.
- N/A Make sure the copyright notice on any files you modified is up to date.
- [x] After creating a pull request, note it in the CHANGELOG.md file
- N/A If preparing for a new release, update the version in cmake.

@dholladay00

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 ## Current develop
 
 ### Fixed (issue #227)
-- [[PR228]](https://github.com/lanl/singularity-eos/pull/228) added untracked header files in cmake
+- [[PR228]](https://github.com/lanl/singularity-eos/pull/228)
+- [[PR229]](https://github.com/lanl/singularity-eos/pull/229)
```
Comment (Yurlungur) [inline CHANGELOG.md:7]:
```suggestion
- [[PR228]](https://github.com/lanl/singularity-eos/pull/228) and [[PR229]](https://github.com/lanl/singularity-eos/pull/229) added untracked header files in cmake
```

**Review by Yurlungur — APPROVED** [review]
Thanks for fixing this.

**Comment by Yurlungur** [comment]
@dholladay00 do you want to take a look before I merge?

**Comment by dholladay00** [comment]
This looks great!

**Review by dholladay00 — APPROVED** [review]
Excellent fixing the issue and cleanup.

**Pull request merged**